mobile/Features/Other/Screens/Options.jsx

```diff
@@ -32,7 +75,20 @@ export default function Options({ navigation }) {
           </View>
         </View>
       ))}
-    </View>
+      {/* <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+        <Button title="Pick an image from camera roll" onPress={pickImage} />
+        {image && <Image source={{ uri: image }} style={{ width: 200, height: 200 }} />}
+      </View> */}
```

Comment:
probably should get deleted but since its the only thing i'm gonna go ahead and merge it! In the next iteration of aws, delete if unnecessary